### ISSUE TITLE
Adafruit Qualia RGB + (TL021WVC02CT / TL040HDS20CT) = strange tearing/flickering even when not writing to the display

### ISSUE THREAD

**Issue by aremmell** [moononournation/Arduino_GFX#407]
**TL;DR:** I am using `Arduino_XCA9554SWSPI`, `Arduino_ESP32RGBPanel`, and `Arduino_RGB_Display` running on an [Adafruit Qualia RGB](https://www.adafruit.com/product/5800) which is an ESP32-S3 board with a 40-pin connector for touch-capable displays such as the [TL021WVC02CT-B1323B](https://www.adafruit.com/product/5792) and [TL040HDS20CT-B1502A](https://www.adafruit.com/product/5794).

I have not experienced issues with this setup in the past, but today both types of display are behaving the same way: in my program's rendering loop, even if I do not write any pixels to any of the above classes (!), there is a visible upward-moving (cycles back to the bottom of the screen) distortion pattern [video](https://rml.dev/pub/qualia-ghost-artifacting.mov). If I draw random magenta rectangles after doing a `fillScreen(BLACK)`, the artifacting/tearing is even more visible [video](https://rml.dev/pub/qualia-magenta-rects-artifacting.mov).

I would really like to use your library to render my graphics on these particular displays because they are my nicest and highest resolution touch displays.

Do you have any clues where to start diagnosing something like this? Have you ever seen it before?

[Edit] I forgot to mention that I am using version 1.4.1 w/ PlatformIO and arduino-espressif32 / esp-idf.

**Comment by moononournation** [comment]
RGBPanel have this known issue, look forward future arduino-esp32 version can improve it.
